### DISCUSSION TITLE
dev-containers: bash: line 1: /workspaces/keda/bin/mockgen: cannot execute binary file: Exec format error

### DISCUSSION THREAD

**Discussion by andrewhibbert** [kedacore/keda#4547] [Q&A / Need Help]
I'm trying to build this locally and getting this error when using dev-containers:

```
bash: line 1: /workspaces/keda/bin/mockgen: cannot execute binary file: Exec format error
```

I'm using a mac which is x86_64

**Comment by JorTurFer** [comment]
This sometimes happens when there are changes related with golang version or binary versions. Remove the `bin` folder (`/workspaces/keda/bin`) and try again
